kindelia_core/src/node.rs

```diff
@@ -1666,8 +1671,11 @@ impl<C: ProtoComm, S: BlockStorage> Node<C, S> {
       eprintln!("  - {}", peer.address);
     }
 
-    eprintln!("Loading block from disk...");
-    self.load_blocks();
+    eprintln!("Loading blocks from disk...");
+    if let Err(e) = self.load_blocks() {
+      eprintln!("Error loading blocks from disk.\n{}", e);
+      // todo:  should we return?
```

Comment:
I don't think we should return. If the loading of blocks from the system has failed, the nodes can still continue to function and will request the missing blocks over the network.

Reply:
Ok, I've removed the comments.

In my mind, there is the related question of the distinction between Node::main() and real main().   Presently Node::main() does not propagate any errors up and just handles/logs them.

I think for now this is ok.  To me, it would really make the most sense to separate the server and cli into two seperate processes anyway.    separation of concerns.  This what what bitcoin has finally done, monero did from the start, etc.  Probably eth as well, but it's been so long since I ran a node I can't remember.

I will file an issue for this later so we can discuss further.